src/interfaces/permissions/request/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { Authorization, JsonMessage } from '../../../messages/types';
+import type { Conditions, Scope } from '../types';
+
+export type PermissionsRequestDescriptor = {
+  description: string
+  grantedTo: string
+  grantedBy: string
+  method: 'PermissionsRequest'
+  scope: Scope
+  conditions: Conditions
```

Comment:
Consider: `GrantScope` and `GrantCondition` else seems to generic of a term.

Reply:
done: https://github.com/TBD54566975/dwn-sdk-js/commit/f99c98c46a5aa494a8041bea0cba32d754c3030a